client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -170,7 +160,10 @@ const MetadataEditor = createClass({
 			</div>
 			*/}
 
-			{this.renderAuthors()}
+			<StringArrayEditor label='authors' valuePatterns={[/^$/]} canEdit={(value)=>value !== global.account?.username && value !== this.props.metadata.authors[0]}
```

Comment:
For `canEdit`, would it be any cleaner to simply provide an array of strings to check against rather than a whole function which essentially has those same strings hardcoded in?